src/pysdmx/model/message.py

```diff
@@ -30,3 +35,24 @@ class ActionType(Enum):
     Replace = "replace"
     Delete = "delete"
     Information = "information"
+
+
+class Header(Struct, frozen=True, kw_only=True):
+    """Header for the SDMX-ML file."""
+
+    id: str = "test"
+    test: str = "true"
+    prepared: datetime = datetime.strptime("2021-01-01", "%Y-%m-%d")
+    sender: str = "Unknown"
+    receiver: str = "Not_Supplied"
+    source: str = "PySDMX"
+    dataset_action: str = ActionType.Information.value
```

Comment:
Receiver, source and dataset action are all with a minOccurs=0 in the SDMX-ML spec. Therefore, we could make them optional and set their default value to None?

Reply:
Yes, also adding a post_init check to ensure the dataset_action is one of the ActionType values.

Reply:
Edited as optional on all values and set type of dataset_action as Optional[ActionType]